win32ss/pch.h

```diff
@@ -8,6 +6,9 @@
  * PROGRAMMER:      Alex Ionescu (alex@relsoft.net)
  */
 
+#ifndef __W32K_H
+#define __W32K_H
+
```

Comment:
I do see no benefit of that move, except that it slows down compile time by a few nanoseconds. A superfluous change.

Reply:
There was no actual change in this header except for moving the info comment block at the top of the file, as we always have.